curriculum/challenges/arabic/14-responsive-web-design-22/build-a-personal-portfolio-webpage-project/build-a-personal-portfolio-webpage.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: bd7158d8c242eddfaeb5bd13
-title: ابن معرضا لأعمالك الخاصة
+title: ابني معرضا لأعمالك الخاصة (Personal Portfolio)
```

Comment:
Hey @Sboonny~!

Why are we including the English project name in the translation? I don't believe other languages are doing this?

Reply:
They weren't translatable, so I included a reference, because of @Nirajn2311 PR there are no need for that anymore.